src/mlpack/methods/neighbor_search/knn_main.cpp

```diff
@@ -64,10 +64,14 @@ PARAM_INT_IN("k", "Number of nearest neighbors to find.", "k", 0);
 // The user may specify the type of tree to use, and a few parameters for tree
 // building.
 PARAM_STRING_IN("tree_type", "Type of tree to use: 'kd', 'vp', 'cover', 'r', "
-    "'r-star', 'x', 'ball', 'hilbert-r', 'r-plus', 'r-plus-plus'.", "t", "kd");
+    "'r-star', 'x', 'ball', 'hilbert-r', 'r-plus', 'r-plus-plus', 'spill'.",
+    "t", "kd");
 PARAM_INT_IN("leaf_size", "Leaf size for tree building (used for kd-trees, vp "
-    "trees, R trees, R* trees, X trees, Hilbert R trees, R+ trees and R++ "
-    "trees).", "l", 20);
+    "trees, R trees, R* trees, X trees, Hilbert R trees, R+ trees, R++ trees "
+    "and spill trees).", "l", 20);
+PARAM_DOUBLE_IN("tau", "Overlapping size (for spill trees).", "u", 0);
+PARAM_DOUBLE_IN("rho", "Balance threshold (for spill trees).", "b", 0.7);
```

Comment:
To emphasize where these are valid, I might consider writing `only valid for spill trees` instead of just `for spill trees`.